src/app/cube/details/action-panel/action-panel.component.html

```diff
@@ -1,63 +1,68 @@
 <div class="inner">
-  <div *ngIf="!isReleased" class="download-warning" tip="This learning object is pending review and may be downloaded when the review is complete."
+  <div *ngIf="!isReleased && !revisedVersion" class="download-warning" tip="This learning object is pending review and may be downloaded when the review is complete."
     tipLocation="top">
     <i class="fas fa-exclamation"></i>Review Pending
   </div>
-  <div class="btn-group vertical" [ngClass]="{'review-pending': !isReleased}">
-    <button tip="{{ tips.DOWNLOAD_NOW }}" tipLocation="top" tipDisabled="{{hasDownloadAccess || !loggedin}}" [ngClass]="{'disabled': !auth.user || !hasDownloadAccess || downloading}"
-      class="button good" id="download-button" (click)="addToCart(true)">
-      <span *ngIf="hasDownloadAccess">{{!downloading ? 'Download Now' : 'Downloading'}}
-        <span *ngIf="downloading">
-          <i class="fal fa-spinner-third fa-spin"></i>
+  <div *ngIf="revisedVersion === false; else reviewerPanel">
```

Comment:
Keeping in theme with @nvisal1 comment above, this can instead be 
```suggestion
  <div *ngIf="!revisedVersion; else reviewerPanel">
```
Also kudos for using Angular templates here

Reply:
Resolved with Remove ===.